assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -1239,6 +1239,49 @@ const baseSelectors = {
 		return state.moduleAccess[ slug ];
 	},
 
+	/**
+	 * Checks if current user has ownership or access to the given module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  Module slug.
+	 * @return {(boolean|undefined)} `true` if the user has ownership or access.
+	 *                               `false` if the user doesn't have ownership or access.
+	 *                               `undefined` If the state is still being resolved,
+	 */
+	hasModuleOwnershipOrAccess: createRegistrySelector(
+		( select ) => ( state, moduleSlug ) => {
+			const moduleStoreName =
+				select( CORE_MODULES ).getModuleStoreName( moduleSlug );
+
+			if ( moduleStoreName === undefined ) {
+				return undefined;
+			}
+
+			// A store with this name doesn't exist, so the user can't have access to it.
+			// This is either caused by a module not being loaded or an incorrect module
+			// name being used.
+			if ( select( moduleStoreName ) === null ) {
+				return false;
+			}
```

Comment:
There wasn't a test for this behaviour, but it was easy to add one so I did: 32df08712130c346b51203e1064710f73573d5a4